umap/static/umap/js/components/alerts/alert.js

```diff
@@ -0,0 +1,177 @@
+class uMapAlert extends HTMLElement {
+  static info(message, duration = 5000) {
```

Comment:
I thought we find a way without custom events ?

Reply:
Yes, we need these but we can make it less verbose when creating events.